captainhook.json

```diff
@@ -0,0 +1,23 @@
+{
+    "commit-msg": {
+        "enabled": true,
+        "actions": [
+            {
+                "action": "\\CaptainHook\\App\\Hook\\Message\\Action\\Regex",
+                "options": {
+                    "regex": "#(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}(\\([a-z ]+\\))?: [\\w ]+#i",
+                    "error": "Commit message does not follow conventional commit standard - https://www.conventionalcommits.org/en/v1.0.0/"
+                }
+            }
+        ]
+    },
+    "pre-commit": {
+        "enabled": true,
+        "actions": [
+            {
+                "action": "git secrets --pre_commit_hook --",
+                "options": []
+            }
```

Comment:
Can we add another action here to run the unit tests as part of pre-commit?

Reply:
Added the action, had to add the script to composer first.